azure-iot-device/tests/common/pipeline/pipeline_stage_test.py

```diff
@@ -80,8 +80,8 @@ class LocalTestObject(StageRunOpTestBase):
         @pytest.fixture(params=unknown_ops)
         def op(self, request, mocker):
             op = make_mock_op_or_event(request.param)
-            op.callback = mocker.MagicMock()
-            add_mock_method_waiter(op, "callback")
+            op.callbacks.append(mocker.MagicMock())  # TODO: make this simpler
+            # add_mock_method_waiter(op, "callback")
```

Comment:
># add_mock_method_waiter(op, "callback") [](start = 12, length = 40)

If these tests pass without this, then you've removed all consumers of this and you can remove this line completely